packages/ember-data/lib/adapters/json-api-adapter.js

```diff
@@ -62,6 +62,44 @@ export default RESTAdapter.extend({
     return Ember.String.pluralize(dasherized);
   },
 
+  /**
+    By default the JSONAPIAdapter will send each find request coming from a `store.find`
+    or from accessing a relationship separately to the server. If your server supports passing
+    ids as a query string, you can set coalesceFindRequests to true to coalesce all find requests
+    within a single runloop.
+    For example, if you have an initial payload of:
+    ```javascript
+    {
+      post: {
+        id: 1,
+        comments: [1, 2]
+      }
+    }
+    ```
```

Comment:
Do you mind adding a new line after all these code block sections? The markdown parser for the api docs doesn't format these sections correctly without the new line to separate content after the code block.